tq42/organization.py

```diff
@@ -59,28 +63,45 @@ def _get(self) -> OrganizationProto:
         Gets the information about the provided organization
         """
         get_org_request = GetOrganizationRequest(id=self.id)
-        res: OrganizationProto = self.client.organization_client.GetOrganization(
-            request=get_org_request, metadata=self.client.metadata
+        res: OrganizationProto = self._client.organization_client.GetOrganization(
+            request=get_org_request, metadata=self._client.metadata
         )
         return res
 
     @staticmethod
     def from_proto(client: TQ42Client, msg: OrganizationProto) -> Organization:
         """
         Creates organization instance from a protobuf message.
+
+        :meta private:
         """
         return Organization(client=client, id=msg.id, data=msg)
 
-    @handle_generic_sdk_errors
     def set(self) -> Organization:
         """
-        Sets the given organization as the default.
+        Sets the current organization as the default organization.
 
-        https://docs.tq42.com/en/latest/Python_Developer_Guide/Setting_Up_Your_Environment.html#changing-your-workspace-to-a-different-organization-or-project
+        :returns: organization instance
+
+        .. deprecated:: 0.8.1
+           Use :py:func:`set_as_default` instead.
         """
-        project = Project.get_default(client=self.client, organization_id=self.id)
+        warnings.warn(
+            "Use of deprecated function set(). Use set_as_default() instead",
+            DeprecationWarning,
+        )
+        return self.set_as_default()
+
+    @handle_generic_sdk_errors
+    def set_as_default(self) -> Organization:
```

Comment:
same comment as for the project. Let's have a quick chat about this, local and remote defaults might be diverging.